sdk/search/Azure.Search.Documents/src/Generated/Models/Autocompletion.cs

```diff
@@ -15,6 +17,15 @@ public partial class Autocompletion
         /// <param name="queryPlusText"> The query along with the completed term. </param>
         internal Autocompletion(string text, string queryPlusText)
         {
+            if (text == null)
```

Comment:
Small nit - I don't feel particularly strongly about this, but we could consider doing the `text ??= throw new ArgNull();` pattern to condense the generated code a bit.  I don't spend a lot of time reading through it, but these diffs always end up pretty massive.

Reply:
Added to https://github.com/Azure/autorest.csharp/issues/577